cmd/root.go

```diff
@@ -0,0 +1,45 @@
+package cmd
```

Comment:
Can you change the name, and put it in a subdirectory? in the future we will have also a web part. 

so it should be something like:
 - `cmd\tools\cli\cli.go`
 - `cmd\tools\web\web.go`